socorro/external/pubsub/crashqueue.py

```diff
@@ -113,10 +113,6 @@ def __iter__(self):
         ``finished_func`` as the only key in ``kwargs``. The caller should call
         ``finished_func`` when it's done processing the crash.
 
-        This does a single pass through the queues and pulls at most 10 from
-        each queue. It tries twice to pull from the priority queue since it's
-        a priority queue.
-
```

Comment:
This isn't true anymore. This was true of a previous implementation that I didn't go with.